### PR TITLE
Add dns update for gcp

### PR DESCRIPTION
**Pull request by rewiko**

### CODE REVIEW

File: lib/gcp/dns.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		Type:    "A",
+		Ttl:     300,
+		Rrdatas: values,
+	}
+
+	dnsChanges := &dns.Change{
+		Additions: []*dns.ResourceRecordSet{recordSetUdpate},
+	}
+
+	_, err := r.gdns.Changes.Create(project, managedZone, dnsChanges).Context(context.Background()).Do()
```
Comment (totahuanocotl) [inline lib/gcp/dns.go]: If a record already exists, it will get a 4XX error, should we check if it exists before ?

Comment (rewiko) [reply lib/gcp/dns.go]: I don't think so it ill overwrite the ip, but for sure I need to handle error.

File: lib/gcp/dns.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	dnsService, err := dns.New(client)
+	if err != nil {
+		return nil, err
+	}
+	return dnsService, err
+}
+
+func (r *impl) UpdateARecords(project string, managedZone string, name string, values []string) error {
+
+	fqdn := name + "." + managedZone
```
Comment (rewiko) [inline lib/gcp/dns.go]: I'm not sure about depends on the managedZone name, but I think we can name the managedZone as subdomain name.

**Comment by totahuanocotl** [comment]
I can't review it now because I made changes too

**Comment by rewiko** [comment]
close for inactivity

**Pull request closed without merging**